src/pages/office/components/fault-comment/fault-comment.ts

```diff
@@ -81,6 +79,20 @@ export class FaultCommentComponent implements OnChanges {
     this.faultCommentChange.emit(commentedCompetency);
   }
 
+  characterCountChanged(charactersRemaining: number) {
+    this.faultCommentCharsRemaining = charactersRemaining;
+  }
+
+  getCharacterCountText() {
```

Comment:
Feels like these should be helpers as they are in multiple classes

Reply:
It could be, but each component has the option to set individual parameters. So that helper function would have a long list of parameters which wouldn't really solve anything. 

And we used this only 2 places so far. The rule is, when you're about the implement the same thing for the 4th time you could start looking into the generalisation of the solution